SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -93,52 +93,67 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 
 	FString ClientWorkerAttribute = GetOwnerWorkerAttribute(Actor);
 
-	WorkerAttributeSet ServerAttribute = { SpatialConstants::ServerWorkerType };
-	WorkerAttributeSet ClientAttribute = { SpatialConstants::ClientWorkerType };
-	WorkerAttributeSet OwningClientAttribute = { ClientWorkerAttribute };
+	WorkerRequirementSet AnyServerRequirementSet;
+	WorkerRequirementSet AnyServerOrClientRequirementSet = { SpatialConstants::UnrealClientAttributeSet };
 
-	WorkerRequirementSet ServersOnly = { ServerAttribute };
-	WorkerRequirementSet ClientsOnly = { ClientAttribute };
-	WorkerRequirementSet OwningClientOnly = { OwningClientAttribute };
+	WorkerAttributeSet OwningClientAttributeSet = { ClientWorkerAttribute };
+
+	WorkerRequirementSet AnyServerOrOwningClientRequirementSet = { OwningClientAttributeSet };
+	WorkerRequirementSet OwningClientOnlyRequirementSet = { OwningClientAttributeSet };
+
+	const TArray<FName>& ServerWorkerTypes = GetDefault<USpatialGDKSettings>()->WorkerTypes.Array();
+	for (const FName& ServerWorkerType : ServerWorkerTypes)
```

Comment:
This is now `FString`

Reply:
@mattyoung-improbable, why did we change this back to FString? I assume that this was because of the UI in the settings editor. Wouldn't it be better to keep them as Fnames if possible?